### DISCUSSION TITLE
Assistant cant do anything on PC

### DISCUSSION THREAD

**Discussion by Gerndall** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#944] [General]
Hey,
like the title said, i cant let do the assistant do anything on my PC like set a timer or open apps. Is it possible and i have forget steps at the setup or are these features not supported?

thanks for the Application and fairly easy setup tutorial

**Comment by im-coder-lg** [comment]
See, the thing about GAUDC is that it's not official(it's right in the name), so all those features are implemented with loopholes in Google's Assistant API system. It might have worked before(when issue number #671 was introduced) but as you can see in the present, those seem to have been patched up by Google, especially the Personal Results feature.
Mentioning #671 for clarity! Let's hope we get a solution for this soon!

**TL;DR**:
GAUDC isn't official, and loopholes seem to be patched. All of us are waiting _eagerly_ for a fix.

**Reply by im-coder-lg** [reply]
I used to say this before but forgot. The GAUDC Project uses Google's Assistant API. They would limit it(proprietary software), so opening apps and stuff like that isn't technically possible with just the SDK. Even setting timers. I think it uses the system apps for that, and the SDK can't access the system apps, unlike Android, where Google itself is linked deeply to the system.
So, due to the SDK code's limits, what you seek can't be naturally implemented, even though they are actually one of the biggest feature requests in this project, along with stuff like Personal Results, and such.
